dmscripts/generate_database_data.py

```diff
@@ -0,0 +1,48 @@
+import random
+import string
+import os
+from dmapiclient import DataAPIClient
+
+from typing import Optional
+
+USER_ROLES = [
+    'admin',
+    'admin-ccs-category',
+    'admin-ccs-sourcing',
+    'admin-manager',
+    'admin-framework-manager',
+    'admin-ccs-data-controller',
+    'buyer',
+    'supplier',
+]
+
+
+DEFAULT_PASSWORD = os.getenv("DM_DEFAULT_PASSWORD", default=None)
```

Comment:
What happens if `DM_DEFAULT_PASSWORD` is not in the environment?

Reply:
Also, our default non-prod password is no secret (e.g. https://github.com/alphagov/digitalmarketplace-scripts/blob/main/scripts/create_test_users.py#L16). So you could include it here if you wanted to...

Reply:
Ah thanks, I didn't realise it wasn't a secret. I'll just set the value rather than trying to hide it in an environment variable